examples/react/start-workos/src/routes/__root.tsx

```
@@ -1,19 +1,15 @@
 import { Box, Button, Card, Container, Flex, Theme } from '@radix-ui/themes';
-import '@radix-ui/themes/styles.css';
 import { HeadContent, Link, Outlet, Scripts, createRootRoute } from '@tanstack/react-router';
 import { TanStackRouterDevtools } from '@tanstack/react-router-devtools';
 import { Suspense } from 'react';
-import { getAuth, getSignInUrl } from '../authkit/serverFunctions';
+import { getAuth, getSignInUrl } from '@workos/authkit-tanstack-react-start';
+import { AuthKitProvider } from '@workos/authkit-tanstack-react-start/client';
 import Footer from '../components/footer';
 import SignInButton from '../components/sign-in-button';
 import type { ReactNode } from 'react';
+import appCssUrl from '../app.css?url';
```

ESLint

[error] `../app.css?url` import should occur before type import of `react`

(import/order)